packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/PaginationFooter/PageSize.js

```diff
@@ -20,20 +20,27 @@ const PageSize = () => {
 
   return (
     <Row>
-      <Select aria-label="Entries per page" onChange={handleChange} value={pageSize}>
+      <Select
+        label={formatMessage({
+          id: 'components.PageFooter.select',
+          defaultMessage: 'Entries per page',
+        })}
+        onChange={handleChange}
+        value={pageSize}
+      >
         <Option value="10">10</Option>
         <Option value="20">20</Option>
         <Option value="50">50</Option>
         <Option value="100">100</Option>
       </Select>
-      <Box paddingLeft={2}>
+      {/* <Box paddingLeft={2}>
```

Comment:
Still needed?

Reply:
Let's leave it if we are asked to match the design of the page size footer in the select.